### ISSUE TITLE
Error when archiving files

### ISSUE THREAD

**Issue by JasonLing95** [eastgenomics/automated-archiving#23]
Error Message on 1 June
```
Error while retrieving session configuration: AttributeError: 'NoneType'
object has no attribute 'pid'
Using env variable HTTPS_PROXY=##### as proxy
Traceback (most recent call last):
  File "main.py", line 106, in <module>
    archiving_function(
  File "/bin/util.py", line 977, in archiving_function
    detail = project.describe()
  File "/usr/local/lib/python3.8/site-packages/dxpy/bindings/dxproject.py", line 74, in describe
    self._desc = api_method(self._dxid, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/dxpy/api.py", line 984, in project_describe
    return DXHTTPRequest('/%s/describe' % object_id, input_params, always_retry=always_retry, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/dxpy/__init__.py", line 686, in DXHTTPRequest
    raise error_class(content, response.status, time_started, req_id)
dxpy.exceptions.ResourceNotFound: The entity project-GP7pQ704Qz8gKf717GyQgXf7 could not be found, code 404. Request Time=1685612083.4749763, Request ID=1685612083536-904528
```

**Comment by JasonLing95** [comment]
Project doesn't seem to exist on DNAnexus - could have been deleted(?)

**Comment by JasonLing95** [comment]
Fixed in #24
